hoptimator

```diff
@@ -8,5 +8,5 @@ $BASEDIR/hoptimator-cli/build/install/hoptimator-cli/bin/hoptimator-cli \
   sqlline.SqlLine \
   -ac sqlline.HoptimatorAppConfig \
   --verbose \
-  -u "jdbc:hoptimator://fun=mysql" -n "" -p "" -nn "Hoptimator" $@
+  -u "jdbc:hoptimator://fun=mysql;hints=offline.table.name=ads_offline" -n "" -p "" -nn "Hoptimator" $@
```

Comment:
I guess we need to get hints working in the CLI somehow.

Maybe we should add a `!hint foo bar` command to the CLI that invokes `HoptimatorConnection.setHint(foo, bar)`. I think we may need to refactor how hints are passed around tho.

Reply:
(This is fine for now.)